docs/kata-containers.md

```diff
@@ -0,0 +1,48 @@
+# Kata containers support in CRI-RM
+
+Kata containers represent an alternative runtime to Runc containers,
+ and are conceptually defined by a containers which replace their sandbox 
+ with a lightweight Virtual Machine.
+
+In order to enable Kata container in a Kubernetes-CRI-RM stack, both Kubernetes 
+and the Container Runtime Engine need to be aware of the new runtime environment:
+
+* The Container Runtime Engine can only be CRI-o or containerd, and will need to
+ have the runtimes enabled in their configuration files.
+* Kubernetes will also have to be made aware via a "RuntimeClass"  resource
+
+After these prerequesites are satisfied, the configuration file for the target 
+Kata Container, will have to have the flag "SandboxCgroupOnly" to true. As of Kata 2.0 this is the only way Kata containers can with the Kubernetes cgroup naming conventions.
+
+```toml
+...
+# if enabled, the runtime will add all the kata processes inside one dedicated cgroup.
+# The container cgroups in the host are not created, just one single cgroup per sandbox.
+# The runtime caller is free to restrict or collect cgroup stats of the overall Kata sandbox.
+# The sandbox cgroup path is the parent cgroup of a container with the PodSandbox annotation.
+# The sandbox cgroup is constrained if there is no container type annotation.
+# See: https://godoc.org/github.com/kata-containers/runtime/virtcontainers#ContainerType
+sandbox_cgroup_only=true
+...
+``` 
+
+Controllers known to be compatible with Kata and CRI-RM:
+* cri & cgroup controller
+* RDT controller
+
+## Kata useful documentation
+
+Easy installation on pre-existing Kubernetes cluster:
+
+* https://github.com/kata-containers/packaging/blob/master/kata-deploy/README.md#kubernetes-quick-start
+
+
+Starting from scratch:
+
+* Kata installation guide: https://github.com/kata-containers/kata-containers/tree/2.0-dev/docs/install#manual-installation
+* Kata Containers + CRI-o: https://github.com/kata-containers/documentation/blob/master/how-to/run-kata-with-k8s.md
+* Kata Containers + containerd: https://github.com/kata-containers/documentation/blob/master/how-to/containerd-kata.md
+* Kubernetes Runtime Class: https://kubernetes.io/docs/concepts/containers/runtime-class/
+* Cgroup and Kata containers: https://github.com/kata-containers/kata-containers/blob/stable-2.0.0/docs/design/host-cgroups.md
+
+

```

Comment:
These seem to use Windows-/DOS-like CR/LF for newlines. Please remove the CR's (ASCII 0x0d).